src/client/packaging/npm/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "delphi_epidata",
-    "version": "0.1.0",
+    "version": "0.2.7",
```

Comment:
@sgratzl is this suspicious?

Reply:
no matches the package.json version field: https://github.com/cmu-delphi/delphi-epidata/blob/ff51a0a5da87c6f37e18d3591afb747c479fe20c/src/client/packaging/npm/package.json#L5